Earthfile

```diff
@@ -5,16 +5,16 @@ WORKDIR /usr/build
 
 build:
   COPY . .
-  RUN npm install && npm --unsafe-perm run bootstrap
-  RUN npm link ./packages/cypress/dist
+  RUN yarn && yarn run bootstrap && yarn
```

Comment:
second `yarn` run is needed here because the first one can't put symlinks into `node_modules/.bin` as those files don't exist (yet). They are created within `yarn run boostrap` so this second `yarn` acts as a fixup here. I plan to fix this later